scripts/router-dashboard-api-wrapper.py

```
@@ -0,0 +1,98 @@
+                "rx_bytes": rx_bytes,
+                "tx_bytes": tx_bytes,
+                "rx_rate": rx_rate,
+                "tx_rate": tx_rate,
+                "rx_packets": int(self.read_file(stats_path / "rx_packets") or 0),
+                "tx_packets": int(self.read_file(stats_path / "tx_packets") or 0),
+                "rx_errors": int(self.read_file(stats_path / "rx_errors") or 0),
+                "tx_errors": int(self.read_file(stats_path / "tx_errors") or 0),
+            }
+
+        self.send_json(interfaces)
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)